docs/source/release/0.59.0-notes.rst

```diff
@@ -0,0 +1,537 @@
+* PR `#9337 <https://github.com/numba/numba/pull/9337>`_: Revert #8583 which skip tests due to M1 RuntimeDyLd Assertion error (`sklam <https://github.com/sklam>`_)
+* PR `#9341 <https://github.com/numba/numba/pull/9341>`_: Add configuration variable to force llvmlite memory manager on / off (`gmarkall <https://github.com/gmarkall>`_)
+* PR `#9342 <https://github.com/numba/numba/pull/9342>`_: Fix flake8 checks for v6.1.0 (`gmarkall <https://github.com/gmarkall>`_)
+* PR `#9346 <https://github.com/numba/numba/pull/9346>`_: Setting ``NUMBA_CAPTURED_ERRORS=old_style`` will now raise warnings. (`sklam <https://github.com/sklam>`_)
+* PR `#9347 <https://github.com/numba/numba/pull/9347>`_: Make Numba's deprecation warnings subclasses of the builtin ones. (`sklam <https://github.com/sklam>`_)
+* PR `#9351 <https://github.com/numba/numba/pull/9351>`_: Made Python 3.12 support rst note more verbose (`kc611 <https://github.com/kc611>`_)
+* PR `#9352 <https://github.com/numba/numba/pull/9352>`_: Removing object mode fallback from `@jit`. (`stuartarchibald <https://github.com/stuartarchibald>`_)
+* PR `#9353 <https://github.com/numba/numba/pull/9353>`_: Remove `numba.generated_jit` (`stuartarchibald <https://github.com/stuartarchibald>`_)
+* PR `#9356 <https://github.com/numba/numba/pull/9356>`_: Refactor print tests to avoid NRT leak issue. (`stuartarchibald <https://github.com/stuartarchibald>`_)
+* PR `#9357 <https://github.com/numba/numba/pull/9357>`_: Fix a typo in `_set_init_process_lock` warning. (`stuartarchibald <https://github.com/stuartarchibald>`_)
+* PR `#9358 <https://github.com/numba/numba/pull/9358>`_: Remove note about OpenMP restriction in wheels. (`stuartarchibald <https://github.com/stuartarchibald>`_)
+* PR `#9359 <https://github.com/numba/numba/pull/9359>`_: Fix test_jit_module test against objmode fallback. (`stuartarchibald <https://github.com/stuartarchibald>`_)
+* PR `#9360 <https://github.com/numba/numba/pull/9360>`_: AzureCI changes. RVSDG test config should still test its assigned test slice (`sklam <https://github.com/sklam>`_)
+
+Authors:
+~~~~~~~~
+
+* `apmasell <https://github.com/apmasell>`_
+* `DrTodd13 <https://github.com/DrTodd13>`_
+* `esc <https://github.com/esc>`_
+* `gmarkall <https://github.com/gmarkall>`_
+* `guilhermeleobas <https://github.com/guilhermeleobas>`_
+* `hellozee <https://github.com/hellozee>`_
+* `kc611 <https://github.com/kc611>`_
+* `kinow <https://github.com/kinow>`_
+* `kkokkoros <https://github.com/kkokkoros>`_
+* `KrisMinchev <https://github.com/KrisMinchev>`_
+* `munahaf <https://github.com/munahaf>`_
+* `njriasan <https://github.com/njriasan>`_
+* `rjenc29 <https://github.com/rjenc29>`_
+* `saulshanabrook <https://github.com/saulshanabrook>`_
+* `sklam <https://github.com/sklam>`_
+* `SridharCR <https://github.com/SridharCR>`_
+* `stuartarchibald <https://github.com/stuartarchibald>`_
+* `Tialo <https://github.com/Tialo>`_
```

Comment:
```suggestion
```
Remove as https://github.com/numba/numba/blob/d4460feb8c91213e7b89f97b632d19e34a776cd3/docs/source/release/0.58.1-notes.rst#L33 was in 0.58.1